File: backend/src/main/java/com/pCarpet/dto/InstructionDTO.java
package com.pCarpet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class InstructionDTO {
	private int id;
	private int duration;
	private String codex;
	private String nameInstruction;
	private int idTask;
}

File: frontend/src/dto/InstructionDTO.ts
export class InstructionDTO{
    private id: number;
    private name: string;
    private duration: number;
    private gcodeFile: string;
    private idTask: number;

    constructor(id: number, name: string, duration: number, gcodeFile: string, idTask: number){
        this.id = id;
        this.duration = duration;
        this.gcodeFile = gcodeFile;
        this.name = name;
        this.idTask = idTask;
    }
}